FreshRSS/FreshRSS
# Improved: Show file permission information in install process, if the access fails

Inspired by #5134

This PR improves the install process step 2 (PHP + folder check).

Before:
just a information came up, that there are not enough permissions on a `./data` directory.

After:
If the permission check fails it shows more information:
- Owner of the directory
- Group of the directory
- file permissions of the directory
- process user (it was displayed in some languages, but not in all)


![grafik](https://user-images.githubusercontent.com/1645099/234319039-f7184f63-cf3e-4839-9185-8f4fd3fe07a4.png)
1) directory
2) Owner of the directory
3) Group of the directory
4) file permissions of the directory
5) Process owner
6) this directory is not available (because I deleted it ;) )


How to test the feature manually:

1. run a fresh installation
2. have "wrong" access configurations to the folders
3. see the messages

Pull request checklist:
